db/carrier_icons.yml

```diff
@@ -2,11 +2,52 @@
   name: ups
   label: UPS
   group: other
+-
+  name: usps
+  label: USPS
+  group: other
+-
+  name: sendle
+  label: Sendle
+  group: other
+  
+-
+  name: poste-italiane
+  label: PosteItaliane
+  group: other
+-
+  name: mondial-relay
+  label: MondialRelay
+  group: other
 -
   name: fedex
-  label: FedEx
+  label: Fedex
+  group: other
+-
+  name: evri
+  label: Evri
+  group: other
+-
+  name: dpd
```

Comment:
For DPD, I think it has to be horizontal per guidelines: https://www.dpd.com/cz/wp-content/uploads/sites/226/2020/05/DPD_Logo-manu%C3%A1l.pdf
![CleanShot 2024-02-02 at 12 23 15@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/89a0b5a7-6ec5-4a0c-ba49-17ee49ab47cb)

![CleanShot 2024-02-02 at 12 23 35@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/fde7a30b-503f-416d-a01e-ebbd2ee3cf3b)

Any thoughts on providing both the icon and then one with the symbol+logo. maybe dpd_symbol.svg
![CleanShot 2024-02-02 at 12 26 13@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/3b44517c-f153-4b41-a535-1237af8d263a)